tests/test_descriptors_cli.py

```diff
@@ -22,57 +22,65 @@ def test_help():
     assert "Usage: janus descriptors [OPTIONS]" in strip_ansi_codes(result.stdout)
 
 
-def test_descriptors(tmp_path):
+def test_descriptors():
     """Test calculating MLIP descriptors."""
-    log_path = tmp_path / "test.log"
-    summary_path = tmp_path / "summary.yml"
-    out_path = tmp_path / "NaCl-descriptors.extxyz"
-    result = runner.invoke(
-        app,
-        [
-            "descriptors",
-            "--struct",
-            DATA_PATH / "NaCl.cif",
-            "--out",
-            out_path,
-            "--log",
+    out_path = Path("./NaCl-descriptors.extxyz").absolute()
+    log_path = Path("./NaCl-log.yml").absolute()
+    summary_path = Path("./NaCl-summary.yml").absolute()
```

Comment:
Why are these no longer `tmp_path`'d?

Reply:
It's testing the minimum inputs for `descriptors`, so all outputs are their default locations, as we do with the other calculations.

For this PR, the log and summary paths are the main point, as we can't save them to `tmp_path` without specifying their full paths, which means we don't test that they use the same prefix as other outputs.

Having done that, I thought doing the same for `out_path` made sense, for similar reasons.

Reply:
Similarly, I think not having any tests without `tmp_path` is the reason we didn't catch #279